packages/data-encoding/data-encoding.0.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://gitlab.com/nomadic-labs/data-encoding"
+bug-reports: "https://gitlab.com/nomadic-labs/data-encoding/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/data-encoding.git"
+license: "MIT"
+depends: [
+  "ocaml" { >= "4.06" }
+  "dune" { >= "1.7" }
+  "lwt"
+  "ezjsonm"
+  "zarith"
+  "ocplib-json-typed" { = "0.7.1" }
+  "ocplib-json-typed-bson" { = "0.7.1" }
```

Comment:
Are those bounds really so strict?

Reply:
It'd probably be fine with `{ >= "0.7" }`. I'll try it now and I'll push a change if it works.